package.json

```diff
@@ -29,49 +29,55 @@
   },
   "homepage": "https://github.com/Unleash/unleash-client-node",
   "dependencies": {
-    "ip": "^1.1.8",
-    "make-fetch-happen": "^10.2.1",
-    "murmurhash3js": "^3.0.1",
-    "semver": "^7.3.8"
+    "ip": "1.1.8",
+    "make-fetch-happen": "11.1.1",
+    "murmurhash3js": "3.0.1",
+    "semver": "7.5.1"
```

Comment:
why are you locking them down?

Reply:
Good call @SimenB, I do not believe this was an intended change.

Reply:
@SimenB I'll get back to you when we decide what to do about pinning dependencies

Reply:
@SimenB updated in `v4.0.1`

Reply:
thanks!